<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_display">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>显示窗口</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_display">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_displayWindow">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_front">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>45</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>向前</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
    <property name="autoRepeatInterval">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>45</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>向左</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
    <property name="autoRepeatInterval">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>45</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>向右</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
    <property name="autoRepeatInterval">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>45</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>向后</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
    <property name="autoRepeatInterval">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_move">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>行使方向</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_move_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>舵机方向</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>564</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_udpport">
           <property name="text">
            <string>udp端口：</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_udpport</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_udpport">
           <property name="text">
            <string>8887</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_tcpport">
           <property name="text">
            <string>tcp端口：</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_tcpport</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_tcpport">
           <property name="text">
            <string>8883</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_fbl">
           <property name="text">
            <string>分辨率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_fbl">
           <item>
            <property name="text">
             <string>292x219</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>176x144</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>320x240</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>640x480</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_IpAddr">
           <property name="text">
            <string>IP地址：</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_IpAddr</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_IpAddr">
           <property name="text">
            <string>192.168.1.4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_bind">
           <property name="text">
            <string>绑定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_stop_ctu">
           <property name="text">
            <string>暂停/继续</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButton_link">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_unlink">
           <property name="text">
            <string>断开连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ledoffon">
         <property name="text">
          <string>开灯/关灯</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>268</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_camera">
         <property name="text">
          <string>摄像头状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_camera_2">
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_move_status">
         <property name="text">
          <string>行驶状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_move_status2">
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_led">
         <property name="text">
          <string>照明：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_led_2">
         <property name="text">
          <string>OFF</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_status_2">
         <property name="text">
          <string>运行状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string>未连接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_info">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_back_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>向下</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
    <property name="autoRepeatInterval">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_front_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>向上</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
    <property name="autoRepeatInterval">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>向左</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
    <property name="autoRepeatInterval">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>向右</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
    <property name="autoRepeatInterval">
     <number>500</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
